### PR TITLE
List projects

### PR DESCRIPTION
**Pull request by zenhack**

### CODE REVIEW

File: tests/unit/api.py [hunk 1]
```diff
@@ -1184,6 +1184,19 @@ def test_free_nodes(self, db):
...
+    def test_list_projects(self, db):
+        assert json.loads(api.list_projects()) == []
+        api.project_create('anvil-nextgen')
+        assert json.loads(api.list_projects()) == ['anvil-nextgen']
+        api.project_create('runway')
+        api.project_create('manhattan')
+        assert sorted(json.loads(api.list_projects())) == [
+            'anvil-nextgen',
+            'manhattan',
+            'runway',
```
Comment (gsilvis) [inline tests/unit/api.py]: Nice names.

**Comment by gsilvis** [comment]
+1

File: tests/unit/api.py (hunk 1 above)
Comment (zenhack) [inline tests/unit/api.py]: I've been using these everywhere --- did you just notice? :stuck_out_tongue:

**Comment by lokI8** [comment]
+1

**Pull request merged**